### PR TITLE
server-1937-review-support-bundle | added support bundle steps

### PR DESCRIPTION
**Pull request by atulsingh0**
# Description
- Added Steps to generate support bundle 

# Reasons
[SERVER-1937](https://circleci.atlassian.net/browse/SERVER-1937
)
# Content Checklist
Please follow our style when contributing to CircleCI docs. Our style guide is here: https://circleci.com/docs/style/.

Please take a moment to check through the following items when submitting your PR (this is just a guide so will not be relevant for all PRs) 😸:

- [ ] Break up walls of text by adding paragraph breaks.
- [ ] Consider if the content could benefit from more structure, such as lists or tables, to make it easier to consume.
- [ ] Keep the title between 20 and 70 characters.
- [ ] Consider whether the content would benefit from more subsections (h2-h6 headings) to make it easier to consume.
- [ ] Check all headings h1-h6 are in sentence case (only first letter is capitalized).
- [ ] Is there a "Next steps" section at the end of the page giving the reader a clear path to what to read next?
- [ ] Include relevant backlinks to other CircleCI docs/pages.

### CODE REVIEW

**Review by lloydChris — CHANGES_REQUESTED** [review]
Perhaps references to `preflight` should be removed since it does not relate to the support bundle?

File: jekyll/_cci2/server/operator/troubleshooting-and-support.adoc [hunk 1]
```diff
@@ -15,6 +15,34 @@ This document describes an initial set of troubleshooting steps to take if you a
 
 toc::[]
 
+[#generate-support-bundle]
+== Generate Support Bundle
+A support bundle is used by CircleCI engineers to diagnose and fix any issues you are experiencing. They are typically requested when you open a ticket.
+
+To generate a support bundle, You need to follow below steps -
```
Comment (rosieyohannan) [inline jekyll/_cci2/server/operator/troubleshooting-and-support.adoc:22]:
```suggestion
To generate a support bundle, follow the steps below.
```

File: jekyll/_cci2/server/operator/troubleshooting-and-support.adoc [hunk 2]
```diff
@@ -15,6 +15,34 @@ This document describes an initial set of troubleshooting steps to take if you a
...
 toc::[]
 
+[#generate-support-bundle]
+== Generate Support Bundle
+A support bundle is used by CircleCI engineers to diagnose and fix any issues you are experiencing. They are typically requested when you open a ticket.
+
+To generate a support bundle, You need to follow below steps - 
+
+=== Prerequisites
+. First, make sure circleci-server is deployed and you have access to the cluster/namespace through kubectl.
```
Comment (rosieyohannan) [inline jekyll/_cci2/server/operator/troubleshooting-and-support.adoc:25]: @atulsingh0 can we add some commands here that people can copy to make this really straightforward for people?

File: jekyll/_cci2/server/operator/troubleshooting-and-support.adoc [hunk 3]
```diff
@@ -15,6 +15,34 @@ This document describes an initial set of troubleshooting steps to take if you a
...
+
+. Next, install link:https://krew.sigs.k8s.io/docs/user-guide/setup/install/[krew].
+
+. Install preflight and support bundle to your local development machine.
```
Comment (rosieyohannan) [inline jekyll/_cci2/server/operator/troubleshooting-and-support.adoc:29]: I'm not clear on this step - what's 'preflight and support bundle' in this context?

File: jekyll/_cci2/server/operator/troubleshooting-and-support.adoc (hunk 2 above)
Comment (atulsingh0) [reply jekyll/_cci2/server/operator/troubleshooting-and-support.adoc:25]: sure

**Pull request merged**